packages/twenty-server/src/engine/workspace-manager/workspace-sync-metadata/constants/standard-field-ids.ts

```diff
@@ -122,7 +122,7 @@ export const companyStandardFieldIds = {
   opportunities: '20202020-add3-4658-8e23-d70dccb6d0ec',
   favorites: '20202020-4d1d-41ac-b13b-621631298d55',
   attachments: '20202020-c1b5-4120-b0f0-987ca401ed53',
-  events: '20202020-0414-4daf-9c0d-64fe7b27f89f',
+  timelineActivities: '72d5d7d3-8782-446c-a54b-1c25024f55db',
```

Comment:
prefix with 20202020, it makes easy to identifiy seed data

Reply:
missed that one thanks!